contributor-ladder.md

```diff
@@ -0,0 +1,222 @@
+# Contributor Ladder Template
+
+This is a light contributor ladder template for CNCF projects that requires editing before it is ready to use. Read the markdown comments, `<!-- COMMENT -->`, for additional guidance. The raw markdown uses `TODO` to identify areas that require customization.
+
+Particularly, this file provides you with a menu of options for your project.  Most projects have 3-4 defined contributor roles. Which of the roles we've outlined below you will use will be based on what makes sense for the structure and size of your project. Other roles should be deleted, or combined into an adjacent role.
+
+
+<!-- template begins here -->
+
+
+* [Contributor Ladder](##contributor-ladder-template)
+    * [Community Participant](###contributor-participant)
+    * [Contributor](###contributor)
+    * [Organization Member](##organization-member)
+    * [Reviewer](###reviewer)
+    * [Maintainer](###maintainer)
+* [Inactivity](##inactivity)
+* [Involuntary Removal](##involuntary-removal)
+* [Stepping Down/Emeritus Process](##stepping-downemeritus-process)
+* [Contact](##contact)
+
+
+## Contributor Ladder
+
+Hello! We are excited that you want to learn more about our project contributor ladder! This contributor ladder outlines the different contributor roles within the project, along with the responsibilites and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows.  Our project members are happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into lists of three types of things. "Responsibilities" are things that contributor is expected to do. "Requirements" are qualifications a person needs to meet to be in that role, and "Rights" are things contributors on that level are entitled to.
+
+
+### Community Participant
+<!--This role spells out what's expected of general community participants.  This is the lowest
+level of participation in the project, and as such many projects don't bother to
+define it and leave this section out.-->
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: A Community Participant participates in the community and contributes their time, thoughts, etc. Community participants are usually users who have stopped being anonymous and started being active in project discussions.
+
+* Responsibilities:
+    * Must follow the CNCF CoC
+* How users can get involved with the community:
+        * Participating in community discussions
+        * Helping other users
+        * Submitting bug reports
+        * Commenting on issues
+        * Trying out new releases
+        * Attending community events
+        * [TODO: Other examples of non-contribution participation]
```

Comment:
The bullets are not rendering because they are over indented:

```suggestion
* How users can get involved with the community:
    * Participating in community discussions
    * Helping other users
    * Submitting bug reports
    * Commenting on issues
    * Trying out new releases
    * Attending community events
    * [TODO: Other examples of non-contribution participation]
```